src/tree-utils.ts

```diff
@@ -267,6 +267,9 @@ export function* iterate<T>(
   index: number = 0,
   count: number = root.getField(Size),
 ) {
+  if (root.getField(Size) === 0) {
+    return;
+  }
```

Comment:
@spion-h4 this is the part that fixes the creation and iteration of empty yads tree.